filepattern-generator-plugin/tests/test_main.py

```diff
@@ -0,0 +1,87 @@
+
+from pathlib import Path
+import os
+import sys
+import filepattern
+dirpath = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(dirpath, '../'))
+import unittest
+from src.main import *
+import fnmatch
+import pyarrow.feather
+import csv
+
+inpDir = Path(dirpath).parent.joinpath('images')
+outDir = Path(dirpath).parent.joinpath('out')
+pattern='p0{r}_x{x+}_y{y+}_wx{t}_wy{p}_c{c}.ome.tif'
+groupBy=None
+chunkSize=2
+outFormat='csv'
+
+class Test_Filepattern_Generator(unittest.TestCase):
```

Comment:
Thanks for making this change.  

I would probably write another test case where you use feather.  

You have an if check on the format but you are only passing in csv so the feather code won't be hit in your test.

Reply:
I have now added tests for both generated files